src/components/Kv/KvResponsiveImage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<img :src="imageUrl" v-bind="$attrs">
+	<img :src="imageUrl" v-show="isRetina != null">
```

Comment:
In order to pass alt text through, we should either still bind the $attrs, or make an alt prop on KvResponsiveImage and pass it down to the image.

Reply:
There is no need to do `v-bind="$attrs"` on root component elements, as Vue already does this automatically. `v-bind="$attrs"` should only be used when you want to bind the attribute to non root elements.

Reply:
ah right! thanks